app/packages/core/src/components/ColorModal/ShareStyledDiv.ts

```diff
@@ -6,7 +6,7 @@ export const ModalWrapper = styled.div`
   left: 0;
   width: 100%;
   height: 100%;
-  z-index: 100000000 !important;
+  z-index: 1000; // do not set more than 1300 (operator panel)
```

Comment:
We should consider having one authoritative file that contains all z-index values. soon ™️

Reply:
Let's talk about this offline.